### DISCUSSION TITLE
Are old special file formats still supported?

### DISCUSSION THREAD

**Discussion by safetycar** [ImageMagick/ImageMagick#3650] [Help] [answered]
Hi, I'm searching in the newer documentation for this features that appear in old docs https://legacy.imagemagick.org/Usage/files/#special_formats
The question is not if it works, because at least "info:" does since I've been using it.
What I wanted to know is why I find it so difficult to find updated information about this like if it was unsupported.

Have I missed the info while searching?
Is it actually considered old stuff that shouldn't be used?
If so there must something considered a better alternative that does the same?

**Comment by urban-warrior** [comment, marked as answer]
All the formats listed in https://legacy.imagemagick.org/Usage/files/#special_formats are still supported.  All supported formats are described here: https://imagemagick.org/script/formats.php.  If you find the documentation lacking, you can create a pull request for the ImageMagick web pages @ https://github.com/ImageMagick/Website/pulls to improve the write-up for a particular format.

**Reply by safetycar** [reply]
Ok, thanks for the reply.
I was trying to search "info:" and "show:" with the colon included that's why I couldn't find it.
I've found "info" and I guess "show" is expected to be used as "win" now since it doesn't appear.

**Reply by fmw42** [reply]
See https://legacy.imagemagick.org/Usage/files/#show
